recipes/medpy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "medpy" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MedPy-{{ version }}.tar.gz
+  sha256: f8a94937dbb947ab069e767862dc6b86896b153c41ce8ed9369c7d79c0033a88
+  patches:
+  - graphcut_boost_lib.patch
+  - fix-compile.patch
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+    - export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -I${PYTHON_INCLUDE_DIR}"
+    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+    - python setup.py build_ext --include-dirs=%PREFIX%\include --library-dirs=%PREFIX%\lib  # [win]
+    - python setup.py build_ext --include-dirs=$PREFIX/include --library-dirs=$PREFIX/lib    # [not win]
```

Comment:
Wouldn't it make more sense to execute these lines *before* running `pip install`?

Reply:
Sure, changed. I just followed one of the recipes in the link posted by @bollwyvl. Made this change, resolving. 😄